packages/server-wallet/src/test_db.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# 1. recreate db if not exist
+# 2. migrate db to latest
+
+for var in "$@"
+do
+    echo setting "$var"...
```

Comment:
This is probably a bit more clear, right? 

```suggestion
    echo creating and migrating "$var"...
```